Misc/NEWS.d/next/Core and Builtins/2017-09-05-14-19-02.bpo-31353.oGZUeJ.rst

```diff
@@ -0,0 +1,5 @@
+PEP 553 - Add a new built-in called ``breakpoint()`` which calls
```

Comment:
PEP 553 can be written:

```
:pep:`553`
```

to get a direct link to the PEP ;-)

Reply:
Doesn’t docutils auto-linkify text that looks like `RFC 1234` or `PEP 123`?